p4_pdpi/p4_runtime_session.cc

```diff
@@ -142,6 +140,51 @@ std::unique_ptr<P4RuntimeSession> P4RuntimeSession::Default(
       new P4RuntimeSession(device_id, std::move(stub), device_id, role));
 }
 
+absl::Status P4RuntimeSession::Write(const p4::v1::WriteRequest& request) {
```

Comment:
Probably this is the first pr touching this file, other pr's are carrying this change, pls fix.